src/votekit/utils.py

```diff
@@ -339,7 +388,9 @@ def tiebroken_ranking(
     ranking: tuple[frozenset[str], ...],
     profile: Optional[PreferenceProfile] = None,
     tiebreak: str = "random",
-) -> tuple[frozenset[str], ...]:
+) -> tuple[
```

Comment:
You have a `random_seed` parameter in `tiebreak_set`. Shouldn't that parameter be here as well if you intend to pass it? Of course, you can get rid of the parameter entirely by telling the user that to make the election deterministic, they need to set the PYTHONHASHSEED environment variable as well as the numpy seed before doing any of their computation.